uwblueprint/ccbc
# [Feat] Magazine Home Page

## Notion ticket link

[Magazine Home Page](https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=0f0df1f1b7074d2cb19b7eb7652d7d04&p=fe884f5195b74c6db8ba9fcedd83ebeb&pm=c)



## Implementation description
* made home page and categories
* updated review api client to include filters
* updated search page with minAge and maxAge filter

Demo video: https://drive.google.com/file/d/1HTO1MNlF75-9wgWqjPDgSukh-AZh68NL/view?usp=sharing


## Steps to test
1. Go to localhost:3000 
2. Search something and press enter -> should redirect to search results page
3. Press the view more links -> all of them except the featured categories should go to search review page with appropriate urls
4. check responsiveness by changing views



## What should reviewers focus on?
* need to wait for design feedback on search bar colour


## Checklist
